Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -730,6 +748,29 @@ public void ReadXmlBuildingJob(XmlReader reader)
     }
 
     /// <summary>
+    /// Reads the XML building job.
+    /// </summary>
+    /// <param name="reader">The XML reader to read from.</param>
+    public void ReadXmlDeconstructJob(XmlReader reader)
```

Comment:
I don't understand the Job Prototype system well enough to add in something else. It could be done, but I'm not sure of the utility at this point.